serverless.yml

```diff
@@ -36,7 +36,7 @@ functions:
     #   - schedule:
     #       name: lambda-ping-${opt:stage}-5min
     #       description: 'Ping HTTP endpoints every 5 minutes'
-    #       rate: rate(5 minutes)
+    #       rate: cron(1,6,11,16,21,26,31,36,41,46,51,56 * * * ? *)
```

Comment:
I set them a minute forward so that in the event they are triggered early, they will still fall in the 5 minute block, 0-5, 5-10, 10-15, etc.